app/services/google_service/client_service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module GoogleService
+  class ClientService
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36'
+
+    BASE_SEARCH_URL2 = 'https://google.com/search'
+
+    def initialize(keyword, lang = 'en')
```

Comment:
https://nimblehq.co/compass/development/code-conventions/ruby/#methods

We dont always implement this this convention, but I guess here it could be applicable to use `keyword` arguments. IMO public methods which are called from outside the class and have multiple arguments passed are a good case for implementing keyword arguments.
```
  # Public method, use keyword argument.
  def initialize(keyword:, lang: 'en')
      escaped_keyword = CGI.escape(keyword)
      @uri = URI("#{BASE_SEARCH_URL2}?q=#{escaped_keyword}&hl=#{lang}&gl=#{lang}")
  end
```

Reply:
I'll use it as a rule of thumb :) 
I remember earlier in this project I faced this question ` when to use these or not ` so yep, I'll update that :)